src/main/java/seedu/address/model/person/Person.java

```diff
@@ -90,8 +90,8 @@ public void removeOrder(Order order) {
      * @return A {@code Customer} instance with default values for other fields.
      */
     public static Person getGuest(Name n, Phone p) {
-        return new Customer(n, p, new Email(), new Address(), new Information(),
-                new Remark(), Collections.emptySet());
+        return new Customer(n, p, new Email("unfilled@unfilled.com"), new Address("unfilled"), new Information(),
+                new Remark("nil"), Collections.emptySet());
```

Comment:
Apart from name and phone, other information is optional. So is it easier to edit the parsing logic to treat empty string as an empty constructor, for example, address = "" ? new Address() : new Address(address). 

I also realized a potential bug, we might need to add a Customer Tag to the set of tags. Please check the constructor of Customer class.

Reply:
works too just in person class there is alot of null checks using requirenonnull so need remove all those